pkg/v4/resource/chartconfig/create_test.go

```diff
@@ -183,6 +183,7 @@ func Test_ChartConfig_newCreateChange(t *testing.T) {
 		K8sClient:   clientgofake.NewSimpleClientset(),
 		Logger:      microloggertest.New(),
 		ProjectName: "cluster-operator",
+		Provider:    "aws",
```

Comment:
In another PR I created `ProviderXYZ` labels in `label` package: https://github.com/giantswarm/cluster-operator/pull/132/files#diff-5e9724242fffab7ce692f8161cb209bc

Maybe take that and use here?